.github/workflows/ci.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
I would prefer this to be a fixed version so it doesn't fall over should GitHub decide to bring a new Ubuntu version on board.